### ISSUE TITLE
Hotkey for bonera doesnt work on blender 4.1 (lattice)

### ISSUE THREAD

**Issue by lucianomunoz** [BlenderBoi/Bonera#5]
Hey, thanks for updating and making this so quick, I dont know for some reason the shift ctrl a hotkey for calling bonera toolkint doesnt work for me in the lattice edit mode, any leads?

Also the addon buttons dont appear

![image](https://github.com/BlenderBoi/Bonera/assets/5134226/8345985f-74ce-4281-a63a-570ad723cfe1)

**Comment by BlenderBoi** [comment]
When You Updating the addon, do you remove it before installing it? Very often addon will bugged out if you update it without removng it first

**Comment by lucianomunoz** [comment]
Yes, I did.

but restarting blender fixed it!

Luciano A. Muñoz Sessarego

*Character Animator*

Web: www.pintamonos.cl

Email: ***@***.***

Imdb: www.imdb.com/name/nm8355386/

Insta: www.instagram.com/lucianomunoz

Linkedin: www.linkedin.com/in/lucianomunoz

Youtube: www.youtube.com/theadventuresoflollypopman

**Comment by BlenderBoi** [comment]
Perfect, is there still any bugs?

**Comment by lucianomunoz** [comment]
Nothing so far,

I was wondering if maybe in the options you could add vertex weights as an
option, you cant autoweight lattices to bones, but you can create vertex
groups and assign weights to bones just like you'd do with a mesh (instead
of having 1000 hook modifiers)

But no bugs for me yet!
stellar work man!

Luciano A. Muñoz Sessarego

*Character Animator*

Web: www.pintamonos.cl

Email: ***@***.***

Imdb: www.imdb.com/name/nm8355386/

Insta: www.instagram.com/lucianomunoz

Linkedin: www.linkedin.com/in/lucianomunoz

Youtube: www.youtube.com/theadventuresoflollypopman

**Comment by BlenderBoi** [comment]
Interesting Idea, It is definitely possible, but probably need to take longer to implement

**Comment by lucianomunoz** [comment]
Yeah, well if you can consider it for your next version itd be great, no
rush
its also great when youre experimenting for instance now i did it with
hooks but then I wanted to merge that armature ton an other one so the
hooks lost their connection, the workaround was to create the bones with
the hooks but telling it to do it in a target armature, if it would have
been with weights attached to armature i could have just pointed the
armature modifier to the new armature.
Either way its already a great addon , thank you!

Luciano A. Muñoz Sessarego

*Character Animator*

Web: www.pintamonos.cl

Email: ***@***.***

Imdb: www.imdb.com/name/nm8355386/

Insta: www.instagram.com/lucianomunoz

Linkedin: www.linkedin.com/in/lucianomunoz

Youtube: www.youtube.com/theadventuresoflollypopman

**Comment by BlenderBoi** [comment]
This is abit hidden but you can actually add the bones to existing armature
<img width="284" alt="image" src="https://github.com/BlenderBoi/Bonera/assets/79613445/2d331b98-5256-4216-a7f6-c427e98b950e">

**Comment by lucianomunoz** [comment]
yeah that was the workaround i mentioned

Luciano A. Muñoz Sessarego

*Character Animator*

Web: www.pintamonos.cl

Email: ***@***.***

Imdb: www.imdb.com/name/nm8355386/

Insta: www.instagram.com/lucianomunoz

Linkedin: www.linkedin.com/in/lucianomunoz

Youtube: www.youtube.com/theadventuresoflollypopman
